PYME/Acquire/Hardware/Piezos/piezo_c867.py

```diff
@@ -237,11 +247,17 @@ def _Loop(self):
                 
                 if not self.errCode == 0:
                     #print(('Stage Error: %d' %self.errCode))
-                    logger.error('Stage Error: %d' %self.errCode)
+                    if _has_gcserr:
+                        logger.error('Stage Error: %s' % gcserr.GCSError.translate_error(self.errCode))
+                    else:
+                        logger.error('Stage Error: %d' % self.errCode)
                 
                 #print self.targetPosition, self.stopMove
                 
-                if self.stopMove:
+                if self.stopMove: # SERVOCHECK: check if this is ok to process when servo is off!!!
+                    # note that issueing the HLT command sets an error condition, from the manual: "Error code 10 is set."
+                    # question: should we check the error status to unset the error code from such a HLT command?
+                    logger.debug('piezo_c867T: stopMove issuing HLT command, expect error 10 to be set')
```

Comment:
This will spam the console with warning messages whenever a soft-joystick (e.g. 3D mouse etc ...) is used.

Reply:
Although I think some of the existing debug messages will as well ...